hoomd/filter/tags.py

```diff
@@ -0,0 +1,26 @@
+from hoomd.filter.filter_ import ParticleFilter
+from hoomd._hoomd import ParticleFilterTags
+import numpy as np
+
+
+class Tags(ParticleFilter, ParticleFilterTags):
```

Comment:
Needs a docstring.

Reply:
Many of them do. I was not sure whether, I would include documentation in this pull request. We have been moving fast trying to get features ported or added rather than the documentation and testing to some extent right now.